packages/frontend/src/main/pages/stream/TheStream.vue

```diff
@@ -57,7 +57,10 @@ export default {
       }
     },
     user: {
-      query: MainUserDataQuery
+      query: MainUserDataQuery,
+      skip() {
```

Comment:
these _other_ missing skips were turning the console red for non-loggedin users cc @fabis94